Improve documentation (#30)
EESSI/compatibility-layer: README.md

After starting the bootstrap have a long coffee...

### Adding the EESSI overlay and packages
Additional packages are added in the EESSI overlay, which is based on ComputeCanada.
You can add them manually or in an automated way by using Ansible, being Ansible the preferred way. Below you can find the two options explained.

#### Ansible playbook (Option 1)
The installation of the EESSI-specific parts can be automatically executed by running the Ansible playbook `install.yml` inside the folder `playbooks`. 

> This playbook will install the (EESSI Gentoo overlay)[https://github.com/EESSI/gentoo-overlay] and a set of packages, including `Lmod` and `archspec`. See the `README` in the `playbooks` folder for more details.

Reviewer: Oops, sorry, I messed up the URL...

```suggestion
This playbook will install the [EESSI Gentoo overlay](https://github.com/EESSI/gentoo-overlay) and a set of packages, including `Lmod` and `archspec`. See the `README` in the `playbooks` folder for more details.
```

### Adding EESSI overlay
Additional packages are added in the EESSI overlay, which is based on ComputeCanada.

> You can add them manually or in an automated way by using ansible. Below you can find the two options explained.

Reviewer: ```suggestion
You can add them manually or in an automated way by using Ansible. Below you can find the two options explained.
```

> ### Adding EESSI overlay

Reviewer: While looking at your changes, I also noticed that the title of this section does not really fit anymore: it does more than just adding the overlay. So maybe this fits better?

```suggestion
### Adding the EESSI overlay and packages
```

emerge --sync
```

#### Ansible playbook (Option 2)

The steps described above can be automatically executed by running the Ansible playbook `install.yml` inside the folder `playbooks`. 

> See the `README` in that folder for more details.

Reviewer: ```suggestion
See the `README` in that folder for more details. Note that the playbook will also install a set of packages, e.g. `Lmod` and `archspec`.
```

Reviewer: I added one more sentence here, to make it clear that the playbook does a little bit more than the described manual procedure.

Author: If those packages do not get installed in the manual installation. Should we add the commands to install them so that one ends in the same scenario?

Reviewer: Yes, agreed. Shall I do that in a separate PR? In that case, just ignore the suggestion about adding this sentence.

Author: I think we can take advantage of this PR to include it. I can try to have a look to the Ansible playbook and do the steps myself.

Reviewer: Okay 👍 
At the moment everything is included in package set `2020.08` (for the current pilot), so it's basically just `emerge @2020.08`, though this will probably change (especially now we're still testing). So, to prevent this from getting outdated very quickly, I'm not sure if we should put the version in the docs. Or maybe just point to [the sets dir in the overlay](https://github.com/EESSI/gentoo-overlay/tree/master/etc/portage/sets) and/or the [defaults file of the playbook](https://github.com/EESSI/compatibility-layer/blob/master/playbooks/roles/compatibility-layer/defaults/main.yml) to find out which sets are available / we installed for the current/latest version of our installation?

Author: @bedroge did not see you message and already made a commit.

What do you think about explaining first ansible and then referring to the ansible playblook in the manual installation?

Reviewer: I think that would maybe be a good solution, yes. We could say that Ansible is the preferred method for installing the compatibility layer, and then show the manual procedure in the way you suggest with links/pointers to the playbook?

> After synching the overlay, add the EESSI package set.

Reviewer: ```suggestion
After synchronizing the overlay, add the EESSI package set(s) that you would like to install, e.g. for set `2020.08`:
```
